src/test/java/seedu/lifeasier/commands/ShowNotesCommandTest.java

```diff
@@ -0,0 +1,18 @@
+package test.java.seedu.lifeasier.commands;
+
+import org.junit.jupiter.api.Test;
+import seedu.lifeasier.notes.NoteList;
+import seedu.lifeasier.notes.Note;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ShowNotesCommandTest {
+    private NoteList notes = new NoteList();
+
+    @Test
+    void execute() {
```

Comment:
Your test names I think you can look at the naming scheme on the module website - SUT name_Inputs_Expected outcome - style